eng/Versions.props

```diff
@@ -109,7 +109,6 @@
     <MicrosoftVisualStudioShellPackagesVersion>17.7.35338-preview.1</MicrosoftVisualStudioShellPackagesVersion>
     <VisualStudioProjectSystemPackagesVersion>17.7.58-pre</VisualStudioProjectSystemPackagesVersion>
     <MicrosoftVisualStudioThreadingPackagesVersion>17.7.5-preview</MicrosoftVisualStudioThreadingPackagesVersion>
-    <MicrosoftBuildOverallPackagesVersion>17.7.0-preview-23217-02</MicrosoftBuildOverallPackagesVersion>
     <!-- Roslyn packages -->
```

Comment:
I believe this needs to be fixed as well, to introduce proper `roslyn` flow into `fsharp`, and get rid of `RoslynVersion` property. We've made a similar change in `razor` repo: https://github.com/dotnet/razor/pull/8492

cc @mmitche @MichaelSimons